src/app/routes/_app/rulesets/$rulesetSlug/rulebooks/$rulebookSlug/edit.module.css

```
@@ -18,8 +18,10 @@
   cursor: grabbing;
 }
 
-.nestedTabs :global([data-rail-over="true"]) {
-  opacity: 1;
+.nestedTabs :global([data-rail-drag-placeholder="true"]) {
```

Stylelint (17.14.0)

[error] Unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)